doc/cli.markdown

```diff
@@ -164,10 +164,10 @@ Users are encouraged to regularly update the balena CLI to the latest version.
 	- [apps](#apps)
 	- [app &#60;nameorslug&#62;](#app-nameorslug)
 	- [app create &#60;name&#62;](#app-create-name)
-	- [app purge &#60;name&#62;](#app-purge-name)
+	- [app purge &#60;nameorslug&#62;](#app-purge-nameorslug)
 	- [app rename &#60;name&#62; [newname]](#app-rename-name-newname)
```

Comment:
As part of this PR, I think we should also further update `app rename`, as per comment https://github.com/balena-io/balena-cli/pull/2128#discussion_r540316336